arch/AArch64/AArch64Disassembler.c

```diff
@@ -39,161 +39,201 @@
+                                               const void *Decoder);
 static DecodeStatus DecodeVecShiftR16Imm(MCInst *Inst, unsigned Imm,
-		uint64_t Addr, const void *Decoder);
+                                         uint64_t Addr, const void *Decoder);
 static DecodeStatus DecodeVecShiftR16ImmNarrow(MCInst *Inst, unsigned Imm,
-		uint64_t Addr,
-		const void *Decoder);
+                                               uint64_t Addr,
+                                               const void *Decoder);
 static DecodeStatus DecodeVecShiftR8Imm(MCInst *Inst, unsigned Imm,
-		uint64_t Addr, const void *Decoder);
+                                        uint64_t Addr, const void *Decoder);
 static DecodeStatus DecodeVecShiftL64Imm(MCInst *Inst, unsigned Imm,
-		uint64_t Addr, const void *Decoder);
+                                         uint64_t Addr, const void *Decoder);
 static DecodeStatus DecodeVecShiftL32Imm(MCInst *Inst, unsigned Imm,
-		uint64_t Addr, const void *Decoder);
+                                         uint64_t Addr, const void *Decoder);
 static DecodeStatus DecodeVecShiftL16Imm(MCInst *Inst, unsigned Imm,
-		uint64_t Addr, const void *Decoder);
+                                         uint64_t Addr, const void *Decoder);
 static DecodeStatus DecodeVecShiftL8Imm(MCInst *Inst, unsigned Imm,
-		uint64_t Addr, const void *Decoder);
-
-static bool Check(DecodeStatus *Out, DecodeStatus In)
-{
-	switch (In) {
-		default:	// never reach
-			return true;
-		case MCDisassembler_Success:
-			// Out stays the same.
+                                        uint64_t Addr, const void *Decoder);
+static DecodeStatus DecodeWSeqPairsClassRegisterClass(MCInst *Inst,
+                                                      unsigned RegNo,
+                                                      uint64_t Addr,
+                                                      const void *Decoder);
+static DecodeStatus DecodeXSeqPairsClassRegisterClass(MCInst *Inst,
+                                                      unsigned RegNo,
+                                                      uint64_t Addr,
+                                                      const void *Decoder);
+static DecodeStatus DecodeSVELogicalImmInstruction(MCInst *Inst,
+                                                   uint32_t insn,
+                                                   uint64_t Address,
+                                                   const void *Decoder);
+static DecodeStatus DecodeSImm(MCInst *Inst, uint64_t Imm,
+                               uint64_t Address, const void *Decoder);
+static DecodeStatus DecodeImm8OptLsl(MCInst *Inst, unsigned Imm,
+                                     uint64_t Addr, const void *Decoder);
+static DecodeStatus DecodeSVEIncDecImm(MCInst *Inst, unsigned Imm,
+                                       uint64_t Addr, const void *Decoder);
+
+static bool Check(DecodeStatus *Out, DecodeStatus In) {
```

Comment:
please put { of a function on the new line, but not at the end

Reply:
be sure to use tabs for indentation, but not spaces

Reply:
but that is how llvm does it :'( 

will fix